src/main/java/io/github/lambdatest/gradle/LambdaTestTask.java

```diff
@@ -93,10 +97,10 @@ public void runLambdaTest() {
         try {
             testExecutor.executeTests(params);
         } catch (IOException e) {
-            System.err.println("Failed to execute tests: " + e.getMessage());
+            logger.error("Failed to execute tests: {}", e.getMessage());
```

Comment:
same here may be!
```suggestion
            logger.error("Failed to execute tests: {}", e);
```

Reply:
I committed your suggestions directly, left the rest as they are. Is there anywhere else I should log the exception fully?

Reply:
no i think.!